tests/postman/ElectionGuard Web Api.postman_collection.json

```diff
@@ -39,7 +39,7 @@
 										},
```

Comment:
I think that this might be a good candidate for also including in the data zips instead of being here in the long run.  No need to change it now, just to keep it in mind for us to discuss.

Reply:
Yea, it is big.  It's also tightly coupled with the API, and I like seeing the diffs and being able to change it in a feature branch (or branches) but not in main.